### ISSUE TITLE
duckdb cli segmentation faults when executing a import tsv and export to parquet (reproduce malloy duckdb imdb example)

### ISSUE THREAD

**Issue by cons0l3** [duckdb/duckdb#5906]
### What happens?

When reproducing the malloy imdb example based on duckdb the duckdb cli seg faults during processing the titles:

```bash
> duckdb < build_titles.sql
 99% ▕███████████████████████████████████████████████████████████▍▏ bash: line 1:   711 Segmentation fault      duckdb < build_titles.sql
```

```bash
> duckdb --version
v0.6.2-dev1114 51945c68c0
```

### To Reproduce

the malloy project examples folder can be found [here](https://github.com/malloydata/malloy/tree/main/samples/duckdb/imdb).

1) download the [build_titles.sql](https://raw.githubusercontent.com/malloydata/malloy/main/samples/duckdb/imdb/build_titles.sql) to a folder
2) download 3 IMDB datasets [title.crew.tsv.gz](https://datasets.imdbws.com/title.crew.tsv.gz), [title.ratings.tsv.gz](https://datasets.imdbws.com/title.ratings.tsv.gz), [title.basics.tsv.gz](https://datasets.imdbws.com/title.basics.tsv.gz) and store them in a subfolder called "data"
3) execute ```duckdb < build_titles.sql```

seg fault, no profit.

### OS:

linux x64, windows 11

### DuckDB Version:

v0.6.1 919cad22e8 and v0.6.2-dev1114 51945c68c0

### DuckDB Client:

cli

### Full Name:

Carsten Steckel

### Affiliation:

Freelancer

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Thanks for the report!

I can verify that this crashes on v0.6.1 - but it seems to work correctly for me on the latest master. The problem seems to be related to writing lists to Parquet files and there was very recently a fix merged for this (#5875).

Could you verify that this is still broken on the latest master?

**Comment by cons0l3** [comment]
Hi @Mytherin,

I had tried on the lastest available version at the time which was v0.6.2-dev1114 51945c68c0. I have just tried the now latest v0.6.2-dev1166 67ceaf6e2c which should include your fix #5875 successfully. 

Case closed ;-)

Thanks
Carsten

**Comment by Mytherin** [comment]
Good news! Thanks for verifying.
